### PR TITLE
Update moose package versions

### LINKED ISSUE #11514: Update packages
## Rationale
Update the versions of moose-environment packages being made available on the MooseDocs documentation site.

## Description
The 'Getting Started' pages needed updating. The manual installation instructions also are becoming stale.

## Impact
None

### PR DESCRIPTION
**Pull request by milljm**
update manual installation instructions, as they were getting
a bit stale.

Removed LLVM instructions from the Manual Installation instructions. I would rather not add that level of complexity to a manual build. GCC is just fine, as most users using the Manual Installation instructions are going to be Linux users (not Darwin users).

Closes #11514

### CODE REVIEW

**Comment by andrsd** [comment]
> Removed LLVM instructions from the Manual Installation instructions.

I sometimes look at those when I build my llvm. Are the gone-gone or can I find them somewhere else?

**Comment by milljm** [comment]
We can create another page supporting an LLVM build I suppose. If not the only other location LLVM/Clang instructions can be found, are in the [package_builder/build_from_source/template/llvm](https://github.com/idaholab/package_builder/tree/master/build_from_source) repo.

**Comment by milljm** [comment]
How and where should we place the following instructions: https://github.com/idaholab/package_builder/blob/master/build_from_source/template/llvm
in our Getting Started pages? Or somewhere else... If anyone has a good spot for these, I will gladly create the documentation page.

**Comment by milljm** [comment]
Do not merge...
Going to add an LLVM support page.

**Comment by brianmoose** [comment]
@milljm  #11518 closed this by accident

**Comment by milljm** [comment]
I still need to run through the instructions for LLVM. There are probably some typos in some of the environmental exports.

File: framework/doc/content/getting_started/installation/manual_installation_gcc.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+application:
+
+```bash
+source /path/to/moose-environment.sh
+```
+
+Or you can permanently have it loaded each time you open a terminal by adding the above `source`
+command in your ~/.bash_profile (or ~/.bashrc which ever your system uses).
+
+!include getting_started/minimum_requirements.md
```
Comment (brianmoose) [inline framework/doc/content/getting_started/installation/manual_installation_gcc.md]: This was already included at the top

**Review by brianmoose — CHANGES_REQUESTED** [review]

**Review by brianmoose — APPROVED** [review]

**Pull request merged**